CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 4.0.2 (Apr 6, 2022)
+
+BUG FIXES:
+
+* Fix typos in `artifactory_federated_*_repository` resources documentation.
```

Comment:
Sorry to be a pain, can you include the PR number (like the other entries)?

(Yes, I just noticed there's a missing one for 4.0.0)